Parquet/Parquet.gradle

```diff
@@ -12,33 +16,19 @@ sourceSets {
     }
 }
 
-
 dependencies {
-    compile project(':ParquetHadoop'),
-            project(':CompilerTools')
+    api project(':ParquetHadoop')
 
-    compile('org.apache.parquet:parquet-hadoop:1.12.0')
-
-    compile('org.apache.hadoop:hadoop-common:2.10.1') {
-        // do not take any dependencies of this project,
-        // we just want a few classes (Configuration, Path) for
-        // simplified prototyping work, and api compatibility.
-        transitive = false
-        // if we actually need any more of hadoop at runtime,
-        // we can add more jars w/ transitive=false,
-        // or replace transitive=false here w/ more exclusions;
-        // (we want to avoid pulling in netty, loggers, jetty-util, guice and asm).
-
-    }
+    implementation project(':CompilerTools')
 
-    runtime 'commons-collections:commons-collections:3.2.2'
-    runtime 'commons-logging:commons-logging:1.2'
+    runtimeOnly 'commons-collections:commons-collections:3.2.2'
+    //runtimeOnly 'commons-logging:commons-logging:1.2'
```

Comment:
We should move this next to the other runtimeOnly dependencies in ParquetHadoop/build.gradle.

```gradle
    runtimeOnly('com.fasterxml.woodstox:woodstox-core:5.2.1')
    runtimeOnly('org.apache.hadoop.thirdparty:hadoop-shaded-guava:1.1.1')
```